hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAbfsInputStreamStatistics.java

```diff
@@ -399,6 +406,89 @@ public void testActionHttpGetRequest() throws IOException {
     }
   }
 
+  @Test
+  public void testPread() throws IOException {
```

Comment:
I don't see much benefit in duplicating the test here. We can just add the stats assertion in the new tests added.

Reply:
Agree. I moved the stats verify also to new test file